docs/docs/tutorials/learn_more.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: default
+title: Learn More
+parent: Tutorials
+nav_order: 6
+---
+
+# Learn More
+
```

Comment:
Add a section here about "how it works", talking about JSON -> protobuf -> grpc server -> protobuf -> JSON.